test/cypress/integration/assessment.js

```diff
@@ -0,0 +1,36 @@
+    cy.get('.btn-success').click();
+  })
+
+  //View Assessment
+  it("Can view the Assessment form", function() {
+    cy.visit(`${tenant.prefix}/login`)
+    cy.login(tenant);
+    cy.visit(`${tenant.prefix}/Admin/Contents/ContentItems`);
+    cy.get('.float-right').click();
```

Comment:
What is this clicking ? Can we add a better selector or document what this is clicking.